### PR TITLE
add ios-net stations

### LINKED ISSUE #217: Missing stations from the Indian Ocean Solar Network (IOS-net)
Hi Adam, 

We had a few more stations last two years in the IOS-net and I see also some are missing.

There is the list : 

```txt
             alias                                               name  territory altitude  latitude longitude                     doi             minDate             maxDate
4           aurere                                             Aurere La Réunion   603.00 -21.01977  55.42472 10.5281/zenodo.10693606 2023-10-30 12:05:00 2024-07-02 02:37:00
13 edfboisdenefles                         Saint Denis Bois de Nefles La Réunion   337.00 -20.91731  55.47644 10.5281/zenodo.10693525 2012-10-11 06:31:00 2023-02-20 04:19:00
14 edflapossession                                 La Possession Baie La Réunion    13.00 -20.93061  55.32897 10.5281/zenodo.10693552 2012-10-11 06:36:00 2023-05-07 09:26:00
15   edfsaintandre                             Saint Andre Chevrettes La Réunion   200.00 -20.96280  55.62243 10.5281/zenodo.10693536 2012-10-11 06:34:00 2023-02-20 05:30:00
16     edfsaintleu                                   Saint Leu Mazeau La Réunion   232.00 -21.20064  55.30233 10.5281/zenodo.10693559 2012-10-11 07:30:00 2024-07-02 02:38:00
17  edfsaintpierre                              Saint Pierre La Valle La Réunion    67.00 -21.31392  55.45107 10.5281/zenodo.10693567 2012-10-11 06:33:00 2024-07-02 02:39:00
21           marla                                              Marla La Réunion  1593.25 -21.10237  55.43058  10.5281/zenodo.8419976 2023-06-16 07:54:00 2024-07-01 16:06:00
33  saintpaulcarat                                Saint Paul Le Carat La Réunion   100.00 -21.04323  55.25089  10.5281/zenodo.8419974 2022-10-24 14:43:00 2024-07-02 02:40:00
34       uomreduit                     University of Mauritius Reduit  Mauritius   200.00 -20.24497  57.48770 10.5281/zenodo.10693484 2022-12-05 13:58:00 2023-04-25 09:13:00
```

Aurère and Marla are stations into the caldera of Mafate in La Réunion (check google maps). 
I should also add three more stations to Reunion by the end of the year, including one on the volcano.

How do you prefers we proceed ?  I can do a **PR** for you if needed. 

I remark in the CSV file that "Components" is missing for all our stations (CSV file inconsistent).

```error
In fread(ss_url, fill = F, header = T, sep = ",") :
  Stopped early on line 195. Expected 16 fields but found 15. 
```

It is mostly Glo and Dir measures on all the stations. 
URL can be https://galilee.univ-reunion.fr/ for all stations. 

Thanks

### PR DESCRIPTION
**Pull request by IoannisSifnaios**
closes #217

### CODE REVIEW

**Comment by IoannisSifnaios** [comment]
@AdamRJensen I wrote the country as La Reunion (France) on purpose so that the stations are not placed in Europe. However, now the `Continent` column is shown as NaN. 

I guess we could put La Reunion in the `State` column? But then again, the `Continent` would be Europe...
Thoughts?

**Comment by AdamRJensen** [comment]
> @AdamRJensen I wrote the country as La Reunion (France) on purpose so that the stations are not placed in Europe. However, now the `Continent` column is shown as `NaN`. 
> 
> I guess we could put La Reunion in the `State` column? But then again, the `Continent` would be Europe...
> Thoughts?

I think i would add "La Reunion (France)" as a new country in the country dictionary and have it map to Africa.

**Comment by IoannisSifnaios** [comment]
I implemented that, but then I realized that there were already some stations from Reunion:

image

Is suppose we should be consistent. Which naming are you in favor of?

**Comment by AdamRJensen** [comment]
Reunion is the English spelling so that's what we should use

File: solarstations.csv [hunk 1]
```diff
@@ -693,3 +693,12 @@ Miyako,,,Japan,24.737,125.327,50,2004-2018,SKYNET,Hitoshi Irie and Tamio Takamur
...
+Saint Andre Chevrettes,,,La Réunion (France),-20.96280,55.62243,200,2012-2023,IOS-net,,Data can be donwloaded from: doi.org/10.5281/zenodo.10693536,https://galilee.univ-reunion.fr/,Freely,SPN1;UV
+Saint Leu Mazeau,,,La Réunion (France),-21.20064,55.30233,232,2012-,IOS-net,,Data can be donwloaded from: doi.org/10.5281/zenodo.10693559,https://galilee.univ-reunion.fr/,Freely,SPN1;UV
+Saint Pierre La Valle,,,La Réunion (France),-21.31392,55.45107,67,2012-,IOS-net,,Data can be donwloaded from: doi.org/10.5281/zenodo.10693567,https://galilee.univ-reunion.fr/,Freely,SPN1;UV
+Marla,,,La Réunion (France),-21.10237,55.43058,1593.25,2023-,IOS-net,,Data can be donwloaded from: doi.org/10.5281/zenodo.8419976,https://galilee.univ-reunion.fr/,Freely,SPN1;UV
+Saint Paul Le Carat,,,La Réunion (France),-21.04323,55.25089,100,2022-,IOS-net,,Data can be donwloaded from: doi.org/10.5281/zenodo.8419974,https://galilee.univ-reunion.fr/,Freely,SPN1;UV
+University of Mauritius Reduit,,,Mauritius,-20.24497,57.48770,200,2022-2023,IOS-net,,Data can be donwloaded from: doi.org/10.5281/zenodo.10693484,https://galilee.univ-reunion.fr/,Freely,SPN1;UV
```
Comment (AdamRJensen) [inline solarstations.csv:704]:
```suggestion
Aurere,,,Reunion,-21.01977,55.42472,603,2023-,IOS-net,,Data can be donwloaded from: doi.org/10.5281/zenodo.10693606,https://galilee.univ-reunion.fr/,Freely,SPN1;UV
Saint Denis Bois de Nefles,,,Reunion,-20.91731,55.47644,337,2012-2023,IOS-net,,Data can be donwloaded from: doi.org/10.5281/zenodo.10693525,https://galilee.univ-reunion.fr/,Freely,SPN1;UV
La Possession Baie,,,Reunion,-20.93061,55.32897,13,2012-2023,IOS-net,,Data can be donwloaded from: doi.org/10.5281/zenodo.10693552,https://galilee.univ-reunion.fr/,Freely,SPN1;UV
Saint Andre Chevrettes,,,Reunion,-20.96280,55.62243,200,2012-2023,IOS-net,,Data can be donwloaded from: doi.org/10.5281/zenodo.10693536,https://galilee.univ-reunion.fr/,Freely,SPN1;UV
Saint Leu Mazeau,,,Reunion,-21.20064,55.30233,232,2012-,IOS-net,,Data can be donwloaded from: doi.org/10.5281/zenodo.10693559,https://galilee.univ-reunion.fr/,Freely,SPN1;UV
Saint Pierre La Valle,,,Reunion,-21.31392,55.45107,67,2012-,IOS-net,,Data can be donwloaded from: doi.org/10.5281/zenodo.10693567,https://galilee.univ-reunion.fr/,Freely,SPN1;UV
Marla,,,Reunion,-21.10237,55.43058,1593.25,2023-,IOS-net,,Data can be donwloaded from: doi.org/10.5281/zenodo.8419976,https://galilee.univ-reunion.fr/,Freely,SPN1;UV
Saint Paul Le Carat,,,Reunion,-21.04323,55.25089,100,2022-,IOS-net,,Data can be donwloaded from: doi.org/10.5281/zenodo.8419974,https://galilee.univ-reunion.fr/,Freely,SPN1;UV
University of Mauritius Reduit,,,Mauritius,-20.24497,57.48770,200,2022-2023,IOS-net,,Data can be donwloaded from: doi.org/10.5281/zenodo.10693484,https://galilee.univ-reunion.fr/,Freely,SPN1;UV
```

File: country_by_continent.json [hunk 2]
```diff
@@ -1 +1 @@
...
... "North America", "Panama": "North America", "Saint Kitts and Nevis": "North America", "Saint Lucia": "North America", "Saint Vincent and the Grenadines": "North America", "United States": "North America", "Argentina": "South America", "Bolivia": "South America", "Brazil": "South America", "Chile": "South America", "Colombia": "South America", "Ecuador": "South America", "Guyana": "South America", "Paraguay": "South America", "Peru": "South America", "Suriname": "South America", "Trinidad and Tobago": "South America", "Uruguay": "South America", "Venezuela": "South America", "Australia": "Oceania", "Cook Islands": "Oceania", "Federated States of Micronesia": "Oceania", "Fiji": "Oceania", "Kiribati": "Oceania", "Marshall Islands": "Oceania", "Nauru": "Oceania", "New Zealand": "Oceania", "Niue": "Oceania", "Palau": "Oceania", "Papua New Guinea": "Oceania", "Samoa": "Oceania", "Solomon Islands": "Oceania", "Tonga": "Oceania", "Tuvalu": "Oceania", "Vanuatu": "Oceania", "Czech Republic": "Europe", "USA": "North America", "Antarctica": "Antarctica", "Reunion": "Africa", "American Samoa": "Oceania", "Canary Islands": "Africa", "Greenland": "North America", "La Réunion (France)": "Africa"}
```
Comment (AdamRJensen) [inline country_by_continent.json:1]:
```suggestion
{"Algeria": "Africa", "Angola": "Africa", "Benin": "Africa", "Botswana": "Africa", "Burkina Faso": "Africa", "Burundi": "Africa", "Cameroon": "Africa", "Cape Verde": "Africa", "Central African Republic": "Africa", "Chad": "Africa", "Comoros": "Africa", "Democratic Congo": "Africa", "Congo": "Africa", "Djibouti": "Africa", "Egypt": "Africa", "Equatorial Guinea": "Africa", "Eritrea": "Africa", "Eswatini": "Africa", "Ethiopia": "Africa", "Gabon": "Africa", "Gambia": "Africa", "Ghana": "Africa", "Guinea": "Africa", "Guinea-Bissau": "Africa", "C\u00f4te d'Ivoire": "Africa", "Kenya": "Africa", "Lesotho": "Africa", "Liberia": "Africa", "Libya": "Africa", "Madagascar": "Africa", "Malawi": "Africa", "Mali": "Africa", "Mauritania": "Africa", "Mauritius": "Africa", "Morocco": "Africa", "Mozambique": "Africa", "Namibia": "Africa", "Niger": "Africa", "Nigeria": "Africa", "Rwanda": "Africa", "S\u00e3o Tom\u00e9 and Pr\u00edncipe": "Africa", "Senegal": "Africa", "Seychelles": "Africa", "Sierra Leone": "Africa", "Somalia": "Africa", "South Africa": "Africa", "South Sudan": "Africa", "Sudan": "Africa", "Tanzania": "Africa", "Togo": "Africa", "Tunisia": "Africa", "Uganda": "Africa", "Zambia": "Africa", "Zimbabwe": "Africa", "Afghanistan": "Asia", "Armenia": "Asia", "Azerbaijan": "Asia", "Bahrain": "Asia", "Bangladesh": "Asia", "Bhutan": "Asia", "Brunei": "Asia", "Cambodia": "Asia", "China": "Asia", "Cyprus": "Asia", "East Timor": "Asia", "Georgia": "Asia", "India": "Asia", "Indonesia": "Asia", "Iran": "Asia", "Iraq": "Asia", "Israel": "Asia", "Japan": "Asia", "Jordan": "Asia", "Kazakhstan": "Asia", "North Korea": "Asia", "South Korea": "Asia", "Kuwait": "Asia", "Kyrgyzstan": "Asia", "Laos": "Asia", "Lebanon": "Asia", "Malaysia": "Asia", "Maldives": "Asia", "Mongolia": "Asia", "Myanmar": "Asia", "Nepal": "Asia", "Oman": "Asia", "Pakistan": "Asia", "Palestine": "Asia", "Philippines": "Asia", "Qatar": "Asia", "Saudi Arabia": "Asia", "Singapore": "Asia", "Sri Lanka": "Asia", "Syria": "Asia", "Tajikistan": "Asia", "Taiwan": "Asia", "Thailand": "Asia", "Turkey": "Asia", "Turkmenistan": "Asia", "United Arab Emirates": "Asia", "Uzbekistan": "Asia", "Vietnam": "Asia", "Yemen": "Asia", "Albania": "Europe", "Andorra": "Europe", "Austria": "Europe", "Belarus": "Europe", "Belgium": "Europe", "Bosnia and Herzegovina": "Europe", "Bulgaria": "Europe", "Croatia": "Europe", "Czechia": "Europe", "Denmark": "Europe", "Estonia": "Europe", "Finland": "Europe", "France": "Europe", "Germany": "Europe", "Greece": "Europe", "Hungary": "Europe", "Iceland": "Europe", "Ireland": "Europe", "Italy": "Europe", "Kosovo": "Europe", "Latvia": "Europe", "Liechtenstein": "Europe", "Lithuania": "Europe", "Luxembourg": "Europe", "Malta": "Europe", "Moldova": "Europe", "Monaco": "Europe", "Montenegro": "Europe", "Netherlands": "Europe", "North Macedonia": "Europe", "Norway": "Europe", "Poland": "Europe", "Portugal": "Europe", "Romania": "Europe", "Russia": "Europe", "San Marino": "Europe", "Serbia": "Europe", "Slovakia": "Europe", "Slovenia": "Europe", "Spain": "Europe", "Sweden": "Europe", "Switzerland": "Europe", "Ukraine": "Europe", "United Kingdom": "Europe", "Vatican City": "Europe", "Antigua and Barbuda": "North America", "The Bahamas": "North America", "Barbados": "North America", "Belize": "North America", "Canada": "North America", "Costa Rica": "North America", "Cuba": "North America", "Dominica": "North America", "Dominican Republic": "North America", "El Salvador": "North America", "Grenada": "North America", "Guatemala": "North America", "Haiti": "North America", "Honduras": "North America", "Jamaica": "North America", "Mexico": "North America", "Nicaragua": "North America", "Panama": "North America", "Saint Kitts and Nevis": "North America", "Saint Lucia": "North America", "Saint Vincent and the Grenadines": "North America", "United States": "North America", "Argentina": "South America", "Bolivia": "South America", "Brazil": "South America", "Chile": "South America", "Colombia": "South America", "Ecuador": "South America", "Guyana": "South America", "Paraguay": "South America", "Peru": "South America", "Suriname": "South America", "Trinidad and Tobago": "South America", "Uruguay": "South America", "Venezuela": "South America", "Australia": "Oceania", "Cook Islands": "Oceania", "Federated States of Micronesia": "Oceania", "Fiji": "Oceania", "Kiribati": "Oceania", "Marshall Islands": "Oceania", "Nauru": "Oceania", "New Zealand": "Oceania", "Niue": "Oceania", "Palau": "Oceania", "Papua New Guinea": "Oceania", "Samoa": "Oceania", "Solomon Islands": "Oceania", "Tonga": "Oceania", "Tuvalu": "Oceania", "Vanuatu": "Oceania", "Czech Republic": "Europe", "USA": "North America", "Antarctica": "Antarctica", "Reunion": "Africa", "American Samoa": "Oceania", "Canary Islands": "Africa", "Greenland": "North America"}
```

**Review by AdamRJensen — APPROVED** [review]

**Pull request merged**